src/nidcpower/metadata/functions.py

```diff
@@ -1815,7 +1817,8 @@
                 'type_in_documentation': 'list of str'
             }
         ],
-        'python_name': 'get_channel_names',
+        'python_name': '_get_channel_names',
```

Comment:
is this necessary? won't the nimi-python generator expand it this way?

Reply:
I believe it is necessary, otherwise the generator would expand the method name to '_get_channel_name_from_string' as the key is 'GetChannelNameFromString'.